CHANGELOG.md

```diff
@@ -9,8 +9,17 @@ Please note that the date associated with a release is the date the code
 was committed to the `main` branch. This is not necessarily the date that
 the code was deployed.
 
+
+
 ## [Unreleased]
 
+### Changes Made - 2023-09-22
```

Comment:
Can you please make this heading just "Changed" and remove the date? It's possible that many other changes will be put under this same heading before our next release. The date will only go in beside the version number once there is one. You can see how this will work further down the in the file.